src/value-editor/common/directives/kp-focusable-input.directive.ts

```diff
@@ -0,0 +1,51 @@
+import { IAttributes, IAugmentedJQuery, IParseService, IScope} from 'angular';
+
+/**
+ * @ngdoc directive
+ * @name kpFocusableInput
+ * @module angularjs-value-editor
+ *
```

Comment:
Jeste pls sem pridej dokumentaci toho parametru. Dela se to tak, ze to das jako parametr, ktery se jmenuje stejne jao ta direktiva. A do tabulky popis injectovatelne promenne, v tomto pripade `$api`.

Reply:
ok